### ISSUE TITLE
Remove Random Response to Bots

### ISSUE THREAD

**Issue by jodylecompte** [campDevs/DiscordBot#67]
Should we remove th feature to randomly respond to other bots? This feature won't do very much good for us as there aren't really any other bots; but what about other guilds?

**Comment by pahosler** [comment]
I vote for removal.

**Comment by glynnw** [comment]
Yeah I second (or third). The main point of keeping it was to demonstrate reactions and that you can still do things with vanilla discord.js.

**Comment by joker314** [comment]
At the current time, does it react to itself? We could make it not do that if it does--making the effects invisible to us but existant in other guilds.
